content/e/kpw/2020-02-01-kpw-arena-16.md

```diff
@@ -19,6 +18,8 @@ It was also KPW's last event for more than 18 months, because of the COVID-19 pa
 
 We saw one debuting wrestler: [Piotr "Opol" Opolski](@/w/piotr-opolski.md). Another foreign guest this time was Austrian wrestler Martn Pain.
 
+The full show was available on Pivotshare VOD, but the service has since closed down. The video was paid on Pivotshare, but is free on YouTube.
```

Comment:
```suggestion
The full show was available as a paid video on Pivotshare VOD, but the service has since closed down, and the full show is free to watch on YouTube.
```